spec/views/posts/show_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'User Show', type: :feature do
+  before :each do
+    @user = User.create(name: 'User1', photo: 'https://picsum.photos/200', bio: 'This is a bio')
+    @post = Post.create(title: 'Post1', text: 'This is a post', author_id: @user.id)
+    @comment = Comment.create(text: 'This is a comment', author_id: @user.id, post_id: @post.id)
+  end
+
+  it 'displays the user name' do
```

Comment:
- [ ] You have done a great job with your tests, however, there are a few tests below that you have missed which are required 👇 **These tests are for the User post page**
- [ ] Tests that asserts you can see the post's title.
- [ ] Tests that asserts you can see who wrote the post.
- [ ] Tests that asserts you can see how many likes it has.
- [ ] Tests that asserts you can see the post body.
- [ ] Tests that asserts you can see the username of each commentor.
- [ ] Tests that asserts you can see the comment each commentor left.
- [ ] Tests that asserts you can see how many comments it has.